client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1374,22 +1374,32 @@
                 var DEFAULT_CLASS = 'main-article theme-container';
 
                 scope.themes = authThemes.availableThemes;
-                authThemes.get(scope.key).then(function(theme) {
+                authThemes.get('theme').then(function(theme) {
                     scope.theme = theme;
-                    applyTheme();
+                    if (scope.key === 'theme') {
+                        applyTheme(theme);
+                    }
+                });
+                authThemes.get('proofreadTheme').then(function(theme) {
+                    scope.proofreadTheme = theme;
+                    if (scope.key === 'proofreadTheme') {
+                        applyTheme(theme);
+                    }
                 });
 
-                scope.changeTheme = function(theme) {
-                    scope.theme = theme;
-                    authThemes.save(scope.key, theme);
-                    applyTheme();
+                scope.changeTheme = function(key, theme) {
+                    scope[key] = theme;
+                    authThemes.save(key, theme);
+                    if (scope.key === key) {
+                        applyTheme(theme);
+                    }
```

Comment:
I would also add docstring here for the method, while we are already changing it...